Tools/clinic/clinic.py

```diff
@@ -5194,13 +5194,14 @@ def bad_node(self, node: ast.AST) -> None:
                     # but at least make an attempt at ensuring it's a valid expression.
                     try:
                         value = eval(default)
-                        if value is unspecified:
-                            fail("'unspecified' is not a legal default value!")
                     except NameError:
                         pass # probably a named constant
                     except Exception as e:
                         fail("Malformed expression given as default value "
                              f"{default!r} caused {e!r}")
+                    else:
+                        if value is unspecified:
+                            fail("'unspecified' is not a legal default value!")
```

Comment:
Without this change, the default value in

```
            module os
            os.access
                follow_symlinks: int(c_default='MAXSIZE') = unspecified
```

was being reported as a "malformed expression" in the error message, which isn't really true. (the `try`/`except` above was catching the `ClinicError` raised by the `fail()` call in the `if value is unspecified` branch)

Reply:
Good catch!